modules/foreman_compute_profile.py

```diff
@@ -69,13 +69,13 @@
 try:
     import nailgun.entities
     import ansible.module_utils.ansible_nailgun_cement as cement
-    HAS_NAILGUN_PACKAGE = True
+    HAS_IMPORT_ERROR = False
```

Comment:
This is a constant because it's ALL_CAPS and should be immutable (though Python doesn't enforce that, try `True = False` ;). Make it a normal variable please.

Reply:
You're absolutely right. Fixed it!